src/tiled/brokenlinks.cpp

```diff
@@ -161,8 +194,21 @@ void BrokenLinksModel::refresh()
                     processTileset(tileset);
                 }
             }
+
+            for (TemplateGroup *templateGroup : mapDocument->map()->templateGroups()) {
+                if (!templateGroup->fileName().isEmpty() && !templateGroup->loaded()) {
+                    BrokenLink link;
+                    link.type = TemplateGroupReference;
+                    link._templateGroup = templateGroup;
+                    mBrokenLinks.append(link);
+                } else {
+                    processTemplateGroup(templateGroup);
```

Comment:
This else makes no sense, because it does essentially the same as the code above, just without the check on the file name being empty (which should be there, I guess).